### ISSUE TITLE
Rewording terms to adjust for Sample Repository

### ISSUE THREAD

**Issue by NTaherifar** [AuScope/ckan-docker#125]
Following terms should be changed:

Organization -> Collection
Dataset -> Sample

**Comment by bmotevalli** [comment]
Rewording is not an easy task as we directly do not have access to CKAN's source code to change all texts within the application. There are two ways:

1. Try to override templates: which is quite challenging as both Organization and Dataset are core concepts and they appear in all pages.
2. Translation: CKAN offers i18n translation capability. An alternate way is to translate Organization to Collection. Similarly for Dataset -> Sample

**Comment by bmotevalli** [comment]
Already done for organization.

**Note:** I tried to do this via our igsn extension. Compilation of .po to .mo works fine, but I couldn't figure out how to make CKAN to refer to this .mo file. Tried to use ITranslate interface but no Luck. As a temp solution, I directly copied the compiled file into CKAN's src directory (ie. src/ckan/ckan/i18n/en_AU). BTW, for a manual approach, this also how it is done.
